setup.cfg

```diff
@@ -0,0 +1,3 @@
+[flake8]
+max-line-length=99
+exclude=migrations
```

Comment:
I would suggest to keep the PRs as clean as possible. 👍 for all the commits but I would suggest to open different PRs for flake/setup.cfg and env vars.